src/functions.cpp

```diff
@@ -114,6 +115,13 @@ void writeConfig(const std::string& filename)
     boost::property_tree::write_ini(filename, tree);
 }
 
+void touchFile(const std::filesystem::path& path)
+{
+    FILE* f = fopen(path.c_str(), "ab");
```

Comment:
Don't use C primitives: `std::ifstream` is a better option. We aim to don't use old and legacy code here.